src/controller/user.controller.js

```diff
@@ -0,0 +1,90 @@
+const userService = require('../services/user.service');
+
+const get = (req, res) => {
+  const users = userService.getAll();
+
+  if (!users) {
+    res.send(users);
```

Comment:
The condition here is incorrect. You're checking if `users` does not exist and then trying to send it as a response. You probably meant to check if `users` is empty or null, and then send a 404 status.